src/plugin/PluginManager.php

```diff
@@ -387,6 +388,19 @@ public function loadPlugins(string $directory, ?array $newLoaders = null) : arra
 		return $loadedPlugins;
 	}
 
+	public function resolveLoadedDependency(Plugin $plugin) : int {
```

Comment:
should there be php-doc on this?

Reply:
PocketMine code uses phpdocs on functions only when it is necessary to add information that is not already provided by type hints and return types (for example the type of elements of an array).

Reply:
I will probably add a phpdoc to indicate what the return value means

Reply:
Dylan said to me that it was needed for phpstan so I just figured it's needed on functions too

Reply:
> only when it is necessary to add information that is not already provided by type hints and return types (for example the type of elements of an array).